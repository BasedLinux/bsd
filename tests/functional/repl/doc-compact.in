:l doc-comments.bsd
:doc compact
