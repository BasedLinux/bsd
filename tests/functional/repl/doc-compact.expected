Bsd <bsd version>
Type :? for help.

bsd-repl> :l doc-comments.bsd
Added <number omitted> variables.

bsd-repl> :doc compact
Function `compact`\
  â€¦ defined at /path/to/tests/functional/repl/doc-comments.bsd:27:5


boom
